internal/controllers/clusterctl/config.yaml

```diff
@@ -11,7 +11,7 @@ data:
     providers:
     # Cluster API core provider
     - name:         "cluster-api"
-      url:          "https://github.com/rancher-sandbox/cluster-api/releases/v1.10.0/core-components.yaml"
```

Comment:
Maybe this specific change caused issues with nightly e2e. It seems that we should bump default CAPI version to latest after all providers released a version with support to it. Which is not ideal.